addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -281,12 +331,21 @@ export const editorCommands = {
                     }
                 }
             }
+            // When multiple <p>text1</p><p>text2</p><p>text3</p><p>text3</p> only start
+            // and end were converting to <li>, and rest were pasted as <p> only, convert
+            // remaining <p> to <li> before pasting on list.
+            if (isBlock(nodeToInsert) && currentNode.nodeName === 'LI') {
```

Comment:
```suggestion
            if (nodeToInsert.nodeName !== LI && isBlock(nodeToInsert) && currentNode.nodeName === 'LI') {
```

Reply:
I did not add that because if `nodeToInsert`'s node name would be equal to the `newTagName`, `setTagName` would just return see [this](https://github.com/odoo/odoo/blob/16.0/addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js#L1997).